webapp/src/main/java/com/box/l10n/mojito/entity/TMTextUnitVariant.java

```diff
@@ -71,7 +71,7 @@ public enum Status {
 
     MT_TRANSLATED,
```

Comment:
Could we add documentation for both MT_TRANSLATED and MT_REVIEW_NEEDED - to be consistent with all the other fields here?